package.json

```diff
@@ -13,7 +13,7 @@
     "build:ts": "tsc",
     "build:contracts": "npm run build:typechain:resolver",
     "build:typechain:resolver": "typechain --target ethers-v4 --outDir ethers './node_modules/@ensdomains/resolver/build/contracts/PublicResolver.json'",
-    "build:api_docs": "rm -rf docs/api && typedoc",
+    "build:api_docs": "rm -rf docs/api && typedoc --plugin typedoc-plugin-markdown --hideBreadcrumbs true",
```

Comment:
asides rendering the documentation as markdown, I also added this so we have consistency across our repos as this is also used by `iam-client-lib`